### PR TITLE
QF-1730-Treat-Ofqual-is-the-intended-EQA-provider-as-Ofqual

### PR DESCRIPTION
**Pull request by Jordan-fujitsu**

### CODE REVIEW

File: src/SFA.DAS.Courses.Application/CoursesImport/Services/StandardsImportService.cs [hunk 1]
```diff
@@ -146,5 +147,17 @@ private async Task AuditImport(DateTime timeStarted, int rowsImported)
             var auditRecord = new ImportAudit(timeStarted, rowsImported);
             await _auditRepository.Insert(auditRecord);
         }
+
+        private void  SetEqaProviderName(List<Domain.ImportTypes.Standard> standards)
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.Courses.Application/CoursesImport/Services/StandardsImportService.cs:151]: Would be better called UpdateEqaProviderName to match the other update call which is done above it

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]
This needs some tests to see that the replacement has been made

File: src/SFA.DAS.Courses.Application/CoursesImport/Services/StandardsImportService.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.Courses.Application/CoursesImport/Services/StandardsImportService.cs:151]: Oh and remove the double space

Comment (Jordan-fujitsu) [reply src/SFA.DAS.Courses.Application/CoursesImport/Services/StandardsImportService.cs]: Renamed method and fixed spacing

**Comment by Jordan-fujitsu** [comment]
@ChrisJWoodcock added unit test` Then_EqaProviderName_Is_Updated_If_Ofqual_Is_Intended` which checks eqa provider name is gets updated to Ofqual when originally set to "Ofqual is the intended EQA provider"

**Review by ChrisJWoodcock — DISMISSED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**